client/components/common/BasicThemedModal/index.jsx

```diff
@@ -80,7 +86,13 @@ const BasicThemedModal = ({
               <FontAwesomeIcon
                 icon={faExclamationTriangle}
                 size="lg"
-                color={theme === 'danger' ? '#ce1b4c' : '#fab700'}
+                color={
+                  theme === 'danger'
+                    ? '#ce1b4c'
+                    : theme === 'warning'
+                    ? '#fab700'
+                    : '#2ba51c' // success
```

Comment:
Wonder if the nested ternary `getThemeColor()` could become a function since it is used twice? Would give an opportunity to use a switch or something a bit easier to read.

Reply:
Nice!